src/main/java/homework_7/KittenToCatFunction.java

```diff
@@ -1,6 +1,6 @@
 package homework_7;
 
 @FunctionalInterface
-public interface KittenToCatFunction<Kitten> {
-     Cat grow(Kitten kitten);
+public interface KittenToCatFunction<K extends Kitten> {
+     Cat grow(K k);
```

Comment:
```suggestion
public interface KittenToCatFunction<K extends Kitten, C extends Cat> {
     C grow(K kitten);
```
Opt: That's what I meant